r/src/array__to_vector.cpp

```diff
@@ -599,6 +640,10 @@ std::shared_ptr<Converter> Converter::Make(const ArrayVector& arrays) {
     case Type::DECIMAL:
       return std::make_shared<arrow::r::Converter_Decimal>(arrays);
 
+    // nested
```

Comment:
Is it worth adding support for FixedSizeList?

Reply:
this is new to me, that probably is another issue/pr.

Reply:
yep, but the code will be similar, almost equal, FixedSizeList also export value_offset.